### ISSUE TITLE
Feature: option to coins based in stabilty 

### ISSUE THREAD

**Issue by arnab2001** [arnab2001/Coinverse#106]

**Comment by pooranjoyb** [comment]
Assign me this issue under JWOC. @arnab2001

**Comment by TuhinBanerjee31** [comment]
Would like to give it a try

**Comment by pooranjoyb** [comment]
> Would like to give it a try

Let's do this together, I will fork your repo, will work together and you make the final PR.  What say? @TuhinBanerjee31 Could be a great leaning experience.

**Comment by TuhinBanerjee31** [comment]
Yeah @pooranjoyb , sounds excellent and surely would be a great learning experience.

**Comment by arnab2001** [comment]
Sure , i can assign you both of that's fine

**Comment by arnab2001** [comment]
Go ahead , I'll be there if you guys need any help
